### PR TITLE
chore(license): update LICENSE and README.md

### PR DESCRIPTION
**Pull request by srl295**
https://github.com/unicode-org/.github/issues/6

### CODE REVIEW

**Comment by annebright** [comment]
Looks good except I don't think we need/want the reference to Maven in the ReadMe. Otherwise fine.

**Comment by srl295** [comment]
fixed

**Review by annebright — DISMISSED** [review]

File: LICENSE [hunk 1]
```diff
@@ -1,46 +1,39 @@
-﻿UNICODE, INC. LICENSE AGREEMENT - DATA FILES AND SOFTWARE
+UNICODE LICENSE V3
```
Comment (srl295) [inline LICENSE:1]: @annebright  it did get changed ^

**Comment by srl295** [comment]
> License title did not get changed to new UNICODE LICENSE V3. Also, I did not designate this repo to get the reference to Maven, but do you know something that I do not that suggests we need the Maven reference?

- It did get changed
- The maven reference was an error and was fixed 3 days ago.

**Review by annebright — CHANGES_REQUESTED** [review]
The Maven reference seems to still be there in the ReadMe notice

**Comment by srl295** [comment]
> The Maven reference seems to still be there in the ReadMe notice

I'm not seeing it. If you go to the Files Changed tab this is what it shows me.  

image

What's the exact URL where you're seeing the maven notice?

**Comment by markusicu** [comment]
> > The Maven reference seems to still be there in the ReadMe notice
> 
> I'm not seeing it.

me neither

**Review by annebright — APPROVED** [review]
Well now I am not seeing it either, so approved.

**Pull request merged**